src/scene/materials/standard-material.js

```diff
@@ -1150,6 +1161,7 @@ function _defineMaterialProps() {
     _defineFlag('occludeSpecular', SPECOCC_AO);
     _defineFlag('shadingModel', SPECULAR_BLINN);
     _defineFlag('fresnelModel', FRESNEL_SCHLICK); // NOTE: this has been made to match the default shading model (to fix a bug)
+    _defineFlag('useDynamicRefraction', false);
```

Comment:
Is there an argument for this to default to true?

Reply:
It can be a bit expensive, the dynamic refractions will also change peoples projects (for the better presumably) but might also incur a cost. 

But it's worth discussing if this could indeed be defaulted to true.